packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -42,54 +49,44 @@ const TimeDuration = styled.time`
   margin: 0 ${GEL_SPACING_HLF};
 `;
 
-const IndexAlsosMediaIndicator = styled.span`
-  & > svg {
-    margin: 0;
-  }
-`;
-
 const MediaIndicator = ({
   datetime,
   duration,
   type,
   topStory,
   service,
-  indexAlsos,
-}) =>
-  indexAlsos ? (
-    <IndexAlsosMediaIndicator aria-hidden="true">
+  isInline,
+}) => (
+  <MediaIndicatorWrapper
+    aria-hidden="true"
+    topStory={topStory}
+    service={service}
+    isInline={isInline}
+  >
+    <FlexWrapper>
       {mediaIcons[type]}
-    </IndexAlsosMediaIndicator>
-  ) : (
-    <MediaIndicatorWrapper
-      aria-hidden="true"
-      topStory={topStory}
-      service={service}
-    >
-      <FlexWrapper>
-        {mediaIcons[type]}
-        {duration && datetime && (
-          <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
-        )}
-      </FlexWrapper>
-    </MediaIndicatorWrapper>
-  );
+      {duration && datetime && (
+        <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
+      )}
+    </FlexWrapper>
+  </MediaIndicatorWrapper>
+);
 
 MediaIndicator.propTypes = {
   datetime: string,
   duration: string,
   type: oneOf(['video', 'audio', 'photogallery']),
   topStory: bool,
   service: string.isRequired,
-  indexAlsos: bool,
+  isInline: bool,
```

Comment:
We should include a small update to the README if we're adding or removing props 😃

Reply:
All done 👍